pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -19,7 +19,6 @@
     </ManifestKeyFile>
     <GenerateManifests>true</GenerateManifests>
     <TargetZone>LocalIntranet</TargetZone>
-    <ApplicationManifest>Properties\app.manifest</ApplicationManifest>
```

Comment:
Why does this get removed when it is getting added to other .csproj files? Is it redundant?